src/utils/genesis_block.ts

```diff
@@ -11,11 +11,69 @@
  *
  * Removal or modification of this copyright notice is prohibited.
  */
+import * as crypto from 'crypto';
+import * as fs from 'fs-extra';
+import path from 'path';
+import { codec, Schema } from '@liskhq/lisk-codec';
+import { Block } from '@liskhq/lisk-chain';
+import { GenesisAssetEntry, GenesisBlockGenerateInput } from '../types';
+import { SNAPSHOT_BLOCK_VERSION, SNAPSHOT_TIME_GAP } from '../constants';
 
-import { KVStore } from '@liskhq/lisk-db';
+export const createChecksum = async (filePath: string): Promise<string> => {
+	const fileStream = fs.createReadStream(filePath);
+	const dataHash = crypto.createHash('sha256');
+	const fileHash = await new Promise<Buffer>((resolve, reject) => {
+		fileStream.on('data', (datum: Buffer) => {
+			dataHash.update(datum);
+		});
+		fileStream.on('error', error => {
+			reject(error);
+		});
+		fileStream.on('end', () => {
+			resolve(dataHash.digest());
+		});
+	});
 
-// TODO: Implement with the issue https://github.com/LiskHQ/lisk-migrator/issues/54
-export const createGenesisBlock = async (db: KVStore, outputPath: string): Promise<any> => ({
-	db,
-	outputPath,
-});
+	return fileHash.toString('hex');
+};
+
+export const createGenesisBlock = async (
+	app: any,
+	assets: GenesisAssetEntry[],
+	blockAtSnapshotHeight: Block,
+): Promise<GenesisBlockGenerateInput> => {
```

Comment:
I think this is returning `Block` from SDK?

Reply:
Yes, updated! Thanks